lib/ReplicateClient.js

```diff
@@ -147,7 +147,7 @@ export default class ReplicateClient {
   }
 
   #parseAction(action) {
-    const result = /^(GET|POST) (\/[^\s]+)$/.exec(action);
+    const result = /^(GET|POST|DELETE) (\/[^\s]+)$/.exec(action);
```

Comment:
Did you mean for this one to be in here too?

Reply:
Oops. No. Thanks.